<launch>
    <param name="cmd_vel_topic" type="string" value="cmd_vel"/>
    <node pkg="sentry_serial" type="con_node" name="con_node" output="screen" 
    args="/dev/ttyUSB0">
	
    </node>


</launch>